File: video/src/main/java/com/leory/badminton/video/mvp/model/bean/VideoListBean.java
package com.leory.badminton.video.mvp.model.bean;

import java.util.List;

/**
 * Describe : 视频列表
 * Author : zhuly
 * Date : 2019-05-15
 */
public class VideoListBean {
    private int totalcount;
    private List<VideoBean>list;

    public int getTotalcount() {
        return totalcount;
    }

    public void setTotalcount(int totalcount) {
        this.totalcount = totalcount;
    }

    public List<VideoBean> getList() {
        return list;
    }

    public void setList(List<VideoBean> list) {
        this.list = list;
    }
}

File: video/src/main/java/com/leory/badminton/video/mvp/model/bean/VideoListBean.kt
package com.leory.badminton.video.mvp.model.bean

/**
 * Describe : 视频列表
 * Author : zhuly
 * Date : 2019-05-15
 */
data class VideoListBean(var totalcount: Int = 0, var list: MutableList<VideoBean>? = null)
